schema/bom-1.5.xsd

```diff
@@ -61,9 +73,9 @@ limitations under the License.
                 See https://cyclonedx.org/capabilities/bomlink/
             </xs:documentation>
         </xs:annotation>
-        <xs:restriction base="xs:string">
+        <xs:restriction base="xs:anyURI">
             <!-- part of the pattern is based on `bom.serialNumber`'s pattern -->
-            <xs:pattern value="urn:cdx:([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})|(\{[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}\})/[1-9][0-9]*#.+"/>
+            <xs:pattern value="urn:cdx:[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}/[1-9][0-9]*#.+"/>
```

Comment:
removed the capability of having those curly-braced UUID formats.
did not finf any docs whether they were allowed here and how to note them down as a BOM-Link